### DISCUSSION TITLE
Three Dots Bug

### DISCUSSION THREAD

**Discussion by demiroo** [vercel/community#6385] [Bug Report]
### Page to Investigate

/settings/environment-variables

### Steps to Reproduce

On this page every time i clicking on the three dots the page scroll to the top that is not so productivity. 

on vercel account: /settings/environment-variables

**Comment by amyegan** [comment]
Thanks for reporting this. I wasn't able to replicate the issue on my own account, but I may be looking in the wrong place. Can you share a screenshot or video to show me what you see?

**Reply by demiroo** [reply]
its a small video but you see it 
![Uploading Bildschirmfoto vom 2024-04-03 22-01-16.png…]()

[Bildschirmaufzeichnung vom 2024-04-03, 22-00-08.webm](https://github.com/vercel/community/assets/69108920/9b88a8c3-d93a-4f7f-9754-e74f8aaf614d)

**Comment by demiroo** [comment]
The bug is  (   tabindex="-1"  )  

![code](https://github.com/vercel/community/assets/69108920/3f08c6b1-3f95-41a5-adf0-f711e9195d13)
